Jamulus.pro

```diff
@@ -238,18 +238,53 @@ win32 {
     LIBS += -framework AVFoundation \
         -framework AudioToolbox
 } else:android {
-    ANDROID_ABIS = armeabi-v7a arm64-v8a x86 x86_64
+    # By default build for all the ABIs
+    # ANDROID_ABIS = armeabi-v7a arm64-v8a x86 x86_64
+
+    # get ANDROID_ABIS from environment - passed directly to qmake
+    ANDROID_ABIS = $$getenv(ANDROID_ABIS)
+
+    # Optional: if ANDROID_ABIS is passed as env var to qmake, will override this
+    # !defined(ANDROID_ABIS, var):ANDROID_ABIS = arm64-v8a
+
+    # Experiments show likely better results with Android 10+ devices
+    ANDROID_MIN_SDK_VERSION = 29
+    ANDROID_TARGET_SDK_VERSION = 32
     ANDROID_VERSION_NAME = $$VERSION
-    ANDROID_VERSION_CODE = $$system(git log --oneline | wc -l)
```

Comment:
This should default to `1` and the build chain should pass the correct value:
- `1` for anything but a release build (as we will not publish to Google Store anything but release builds).
- retrieve `last_release_android_version` from permanent storage, add the number of ABI builds to be released plus one, store that in `last_release_android_version` permanent storage and pass it to the build.